lib/form-validation.js

```diff
@@ -6248,6 +6248,11 @@
       fvValidateAtt = "".concat(dataPrefix, "validate-form"),
       fvErrorParentAtt = "".concat(dataPrefix, "validate-error-parent"),
       fvDressedAtt = "".concat(dataPrefix, "formvalidation-dressed"),
+      fvNumericalityOnlyIntegerAtt = "".concat(dataPrefix, "numericality-only-integer"),
+      fvNumericalityGreaterThanAtt = "".concat(dataPrefix, "numericality-greater-than"),
+      fvNumericalityGreaterThanOrEqualToAtt = "".concat(dataPrefix, "numericality-greater-than-or-equal-to"),
+      fvNumericalityLessThanAtt = "".concat(dataPrefix, "numericality-less-than"),
```

Comment:
ah zie opmerking boven. het is er wel maar wordt niet gedemo'd

Reply:
Jawel, lees de demo pagina en e2e testen nog maar eens :)

Reply:
Ja ik zie het nu, het ene bij integer en het ander bij decimalen